dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -31,7 +32,7 @@ internal class RumViewScope(
     internal val firstPartyHostDetector: FirstPartyHostDetector
 ) : RumScope {
 
-    internal val urlName = name.replace('.', '/')
+    internal val url = key.resolveViewUrl().replace('.', '/')
```

Comment:
No, since the `resolveViewUrl` will also provide the default view name. I think it's better for the view name to have the classname as people are used to, and URL in Url format